pkg/network/vmispec/interface.go

```diff
@@ -188,3 +188,13 @@ func HasBindingPluginDeviceInfo(iface v1.Interface, bindingPlugins map[string]v1
 	}
 	return false
 }
+
+func HaveMasqueradeInterface(interfaces []v1.Interface) bool {
```

Comment:
Should the `ISTIO_KUBEVIRT_ANNOTATION` be added only when the pod network is bound with masquerade binding?

Reply:
Masquerade binding is supported only on the pod network. You cannot use it as a secondary.

Reply:
Thank you for the explenation.
Do you maybe remember what is the reason `IsPodNetworkWithMasqueradeBindingInterface` returns `true` when there is no pod network?
https://github.com/kubevirt/kubevirt/blob/4ad0aead6936b6a79c56c09fe923226911584416/pkg/network/vmispec/interface_test.go#L40

Reply:
I think it is something that rolled through several iterations reflecting the original logic. I can only guess that the original assumed that masquerade is the default if nothing is specified.

But I do not think it is correct. It worked so far because the current usage did not call it with an empty slice.